src/Header/Header.js

```diff
@@ -161,6 +173,46 @@ export default function Header(props) {
     props.setDarkTheme();
   };
 
+  function CheckLoggedInAuth0() {
```

Comment:
Inconsistent naming. Non-JSX functions should be camelCased and JSX functions must be PascalCased